src/main/java/spleetwaise/transaction/logic/commands/FilterCommand.java

```diff
@@ -37,6 +38,7 @@ public class FilterCommand extends Command {
                     + "[INDEX] " + "[" + PREFIX_AMOUNT + "AMOUNT] " + "[" + PREFIX_DESCRIPTION
                     + "DESCRIPTION] " + "[" + PREFIX_DATE + "DATE] " + "[" + PREFIX_STATUS
                     + "STATUS] " + "[" + PREFIX_AMOUNT_SIGN + "AMOUNT_SIGN]\n"
```

Comment:
```suggestion
                    + "STATUS] " + "[" + PREFIX_AMOUNT_SIGN + "AMOUNT_SIGN] "
```
Extra `\n` here since you have added the category prefix

Reply:
Thank you